Repositories/UserV1Repository.cs

```diff
@@ -1,6 +1,5 @@
 using rexfinder_api.Models;
 using rexfinder_api.Migrations;
-using bcrypt = BCrypt.Net.BCrypt;
 
```

Comment:
That is also not needed here.  The only place it's needed is the in AuthService.cs.  It's impossible to change the password in UserV1Repository.cs.  I tried changing the password in Postman and it just ignores that property.  It changes everything else except the password. To change the password, I would need to add this: "originalUser.Username = editUser.Password;" Since it's NOT in the code, you can't change it, so no need for the hash.